.github/workflows/test-check.yml

```diff
@@ -0,0 +1,78 @@
+name: Test Checker
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+
+jobs:
+  test_check:
+    name: Check test results
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for build to succeed ( ubuntu )
+        uses: tomchv/wait-my-workflow@v1.1.0
+        id: wait-for-build-ubu
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "Tests & Build (ubuntu-latest)"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 7200
+
+      - name: Make sure the build was a success ( ubuntu )
+        if: steps.wait-for-build-ubu.outputs.conclusion != 'success'
+        run: echo buils not success && false
+
+      - name: Wait for build to succeed ( windows )
+        uses: tomchv/wait-my-workflow@v1.1.0
+        id: wait-for-build-win
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "Tests & Build (windows-latest)"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 7200
+
+      - name: Make sure the build was a success ( windows )
+        if: steps.wait-for-build-win.outputs.conclusion != 'success'
+
+        run: echo buils not success && false
+      - name: Wait for build to succeed ( macos )
+        uses: tomchv/wait-my-workflow@v1.1.0
+        id: wait-for-build-mac
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "Tests & Build (macos-latest)"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 7200
+
+      - name: Make sure the build was a success ( macos )
+        if: steps.wait-for-build-mac.outputs.conclusion != 'success'
+        run: echo buils not success && false
+
+      - name: Sleep to wait for the build to trigger
+        run: sleep 1m
+
+      - name: Wait for test to succeed
+        uses: tomchv/wait-my-workflow@v1.1.0
+        id: wait-for-test
+        if: steps.wait-for-build.outputs.conclusion == 'success'
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: "Integration Tests"
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          timeoutSeconds: 1200
+
+      - run: echo '${{ toJson(steps.wait-for-test) }}'
```

Comment:
This is probably some leftover or development stage of this workflow. Sorry if reviewing too early.

Reply:
you are correct, its removed now!

No worries, i'm very glad with the review comments!